src/core/TimePos.cpp

```diff
@@ -161,11 +161,11 @@ tick_t TimePos::getTickWithinBeat( const TimeSig &sig ) const
 
 f_cnt_t TimePos::frames( const float framesPerTick ) const
 {
-	if( m_ticks >= 0 )
-	{
-		return static_cast<f_cnt_t>( m_ticks * framesPerTick );
-	}
-	return 0;
+	//TODO: Remove next line, it's just a warning to see
+	// if any leftover code is still creating notes with
+	// negative lengths
+	if (m_ticks < 0) { qWarning("TimePos with negative length"); }
```

Comment:
Was going to comment on this in the first review, but I wasn't sure if having it would be necessary or not given the surrounding code. I just noticed you said it can be removed, although I think it could be replaced with an ``assert`` (assuming that negative lengths should never happen)?

Reply:
The idea was to remove this completely before merging, negative length notes should not happen anymore after this change. I can also replace it with an assert, if there's interest in keeping the test here.